hw1/ex1.py

```diff
@@ -0,0 +1,63 @@
+import scipy.io.wavfile as wf
+import sounddevice as sd
+import tensorflow as tf
+import argparse
+import os
+from time import time
+
+def get_audio_from_numpy(indata):
+ indata = tf.convert_to_tensor(indata, dtype=tf.float32)
+ indata = 2*(indata + 32768) / (32767 + 32768) -1 
+ indata = tf.squeeze(indata)
+ return indata
+
+def get_spectrogram(indata, samplerate, frame_length_in_s, frame_step_in_s):
+    audio = get_audio_from_numpy(indata)
+    frame_length = int(frame_length_in_s * samplerate)
+    frame_step = int(frame_length_in_s * samplerate)
```

Comment:
Why don't you use the parameter frame_step_in_s? What if the two input parameters frame length and frame step don't have the same value?